test/client/fixtures/core/jquery-patched-test/iframe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
maybe `sandboxed-jquery-test` instead of `jquery-patched-test`? If we use `sandboxed` in other code